modules/ROOT/pages/database-internals/neo4j-admin-store-info.adoc

```diff
@@ -85,41 +85,52 @@ The `neo4j-admin database info` command has the following options:
 ====
 [source, shell]
 ----
-bin/neo4j-admin database info mygraph.db
+bin/neo4j-admin database info healthcare
 ----
 
-Output:
-
+.Output
+[results]
 ----
-Store format version:         record-aligned-1.1
-Store format introduced in:   5.0.0
+Database name:                healthcare
+Database in use:              false
+Store format version:         block-block-1.1
```

Comment:
If we provide example with the `block` format, should we label it 'Enterprise edition'?

Reply:
Do you mean to label the example output?

Reply:
I was thinking about the labelling the whole example.
Now I think this would look weird.
I'm wondering if we need to clarify that we test Enterprise edition here. For example,  

```
[source, shell]
----
bin/neo4j-admin database info healthcare
----
+
If you're using Neo4j Enterprise Edition, the output can be the following:
+
[results]
----
Database name:                healthcare
Database in use:              false
Store format version:         block-block-1.1
<....>
----
```

Reply:
I could add a sentence before all examples to say, "The following examples assume that you are using Neo4j Enterprise edition."

Reply:
Hey @NataliaIvakina, I added two intro sentences. Could you please take a look if they make sense?

Reply:
Perfect 👌 Looks great!